.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install kubectl
+      run: |
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl
+
+    - name: Install krew
+      shell: bash
```

Comment:
Do you have some reference where these options are documented? I wonder what the implications of `shell: bash` are (I'm not saying this is bad in any way).

Reply:
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#using-a-specific-shell

bash is the best option here.

Reply:
Looks like `set -u` is still required.

Reply:
@corneliusweig I agree with you, it's done :)